settings.gradle

```diff
@@ -3,8 +3,15 @@ rootProject.name = 'alfred-telemetry'
 // Alfred Telemetry core module
 include ':alfred-telemetry-platform'
 
+// Alfred Telemetry core module
+include ':alfred-telemetry-solr'
+["solr4","solr6"].each { solrFlavor ->
+    include ":alfred-telemetry-solr:${solrFlavor}"
+}
+
 include ':integration-tests'
-["52", "60", "61"].each { version ->
-    include ":integration-tests:alfresco-enterprise-${version}"
+//["52", "60", "61"].each { version ->
```

Comment:
Do we want to drop tests agains Alfresco 6.0? And even if we want, it should not be part of this PR.